students/lauraannf/lessons/lesson01/assignment/test_unit.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Mar 12 13:53:15 2019
+
+@author: Laura.Fiorentino
+"""
+
+from unittest import TestCase
+from unittest.mock import Mock, MagicMock, patch
+from inventory_management.market_prices import get_latest_price
+from inventory_management.furniture import Furniture
+from inventory_management.inventory import Inventory
+from inventory_management.electricappliances import ElectricAppliances
+from inventory_management.main import main_menu
+from inventory_management.main import get_price
+from inventory_management.main import add_new_item
+from inventory_management.main import item_info
+from inventory_management.main import exit_program
```

Comment:
It's ok for this project, for larger projects, you might want to do:

`import inventory_management`

Or:

`from inventory_management import main`

The reason is that when a test fails, you'll have to go over all your import to figure out, for example, which module _item_info()_ came from. It would be easier if your call for _item_info()_ is _inventory_management.main.item_info()_.